examples/main/index.ts

```diff
@@ -11,9 +11,12 @@ console.log('main function started');
 // cleanup unused sessions every 30s
 // setInterval(async () => {
 //     const { activeUserWorkersCount } = await EdgeRuntime.getRuntimeMetrics();
+//
+//     /* Since cleanup logic will look into active consumer workers we can avoid this check
```

Comment:
shall we remove this code? It may lead to incorrect implementations, that would never cleanup.

Reply:
You're right, I did remove it